tools/scripts/buildModOPT.bat

```diff
@@ -40,38 +46,38 @@ goto processArgs
 
 :release
 	:: build release
-	call %~dp0\createModDir.bat OPT release
+	call %~dp0.\createModDir.bat OPT release
 	
 	echo Building release version of OPT Mod...
 	
 	:: move away all old PBOs
-	for /f %%a IN ('dir %~dp0\..\..\PBOs\release\@OPT\addons\ /b') do move %~dp0\..\..\PBOs\release\@OPT\addons\%%a %~dp0\..\..\PBOs\archive\release\ >nul
+	for /f %%a IN ('dir %~dp0.\..\..\PBOs\release\@OPT\addons\ /b') do move %~dp0.\..\..\PBOs\release\@OPT\addons\%%a %~dp0.\..\..\PBOs\archive\release\ >nul
 	
-	%~dp0\..\programs\armake2.exe build -i %~dp0\..\..\dependencies\CLib\addons\ -x pboName.h %~dp0\..\..\addons\OPT\ %~dp0\..\..\PBOs\release\@OPT\addons\%pboName%
+	%~dp0.\..\programs\armake2.exe build -i %~dp0.\..\..\dependencies\CLib\addons\ -x pboName.h %~dp0.\..\..\addons\OPT\ %~dp0.\..\..\PBOs\release\@OPT\addons\%pboName%
 
 if not [%version%] == [both] goto finish
 
 
 :dev
 	:: build dev
-	call %~dp0\createModDir.bat OPT dev
+	call %~dp0.\createModDir.bat OPT dev
 	
 	echo Building dev version of the OPT Mod...
 	
 	:: move away all old PBOs
-	for /f %%a IN ('dir %~dp0\..\..\PBOs\dev\@OPT\addons\ /b') do move %~dp0\..\..\PBOs\dev\@OPT\addons\%%a %~dp0\..\..\PBOs\archive\dev\ >nul
+	for /f %%a IN ('dir %~dp0.\..\..\PBOs\dev\@OPT\addons\ /b') do move %~dp0.\..\..\PBOs\dev\@OPT\addons\%%a %~dp0.\..\..\PBOs\archive\dev\ >nul
 	
 	:: in order to build the dev-version the ISDEV macro flag has to be set programmatically
-	1>NUL copy %~dp0\..\..\addons\OPT\isDev.hpp %~dp0\..\..\addons\OPT\isDev.hpp.original
-	echo:>> %~dp0\..\..\addons\OPT\isDev.hpp
-	echo|set /p="#define ISDEV" >> %~dp0\..\..\addons\OPT\isDev.hpp
+	1>NUL copy %~dp0.\..\..\addons\OPT\isDev.hpp %~dp0.\..\..\addons\OPT\isDev.hpp.original
+	echo:>> %~dp0.\..\..\addons\OPT\isDev.hpp
+	echo|set /p="#define ISDEV" >> %~dp0.\..\..\addons\OPT\isDev.hpp

```

Comment:
warum so umständlich?
ist es nötig das der output ohne newline erfolgt?
sonst reicht doch ein simples `echo #define ISDEV >> ...`

Reply:
Öhm - könnte gut sein, dass das noch ein Überbleibsel aus einem alternativen Ansatz ist. Ich sehe spontan keinen Grund, warum das nötig ist.
Kannst es ja mal ohne sonderlösung probieren ^^

Das `echo:>> %~dp0.\..\..\addons\OPT\isDev.hpp` ist aber nötig, um sicherzustellen, dass der danach eingefügte Text in einer eigenen Zeile steht und nicht zufällig irgendwo in nem anderen Preprocessor-Statement geschluckt wird

Reply:
done in #22